src/xas_tdp_atom.F

```diff
@@ -3206,17 +3195,13 @@ END SUBROUTINE integrate_so_prod
 !> \param intso the integral in terms of the spherical orbitals (well, their derivative)
 !> \param V the potential (put on the grid and weighted) to integrate
 !> \param ikind the atomic kind for which we integrate
-!> \param xas_atom_env ...
 !> \param qs_env ...
-!> \note The atomic grids are taken fron xas_atom_env and the orbitals are the normal ones. Ok since
```

Comment:
"fron" => "from"

Reply:
Well, this goes away anyway. I am sure that if you look hard enough, you might find a few more typos in my comments

Reply:
ufff, right, my bad